components/esp32s2/Kconfig

```diff
@@ -368,6 +368,27 @@ menu "ESP32S2-specific"
         # high resolution timekeeping anymore. Instead the esp_timer API, implemented
         # using FRC2 timer, is used.
         # FRC1 name in the option name is kept for compatibility.
+
+    config ESP32_BROWNOUT_SHUTDOWN_IF_EARLY
```

Comment:
probably should rename to `ESP32S2_BROWNOUT_SHUTDOWN_IF_EARLY` for consistency with other Kconfig entries.